src/main/java/core/basesyntax/orderprocessing/ResultWriter.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.orderprocessing;
+
+import com.opencsv.CSVWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class ResultWriter {
```

Comment:
```suggestion
public class FileWriter {
```

Reply:
Can't rename it that way because I use java.io.FileWriter in this class